index.js

```diff
@@ -23,6 +23,7 @@ MarionetteApps.ModuleToFunction = {
   launch: 'launch',
   list: 'list',
   switchtoapp: 'switchToApp',
+  switchtoactivity: 'switchToActivity',
```

Comment:
Basically it's inline activity only, but ```swithToInlineActivity``` sounds too verbose, but don't have strong opinion